src/components/__test__/Nav.test.js

```diff
@@ -0,0 +1,14 @@
+import renderer from 'react-test-renderer';
```

Comment:
- [ ] Since this is a big app, I think you should add more tests which would cover better your application from potential errors

Reply:
I did three test for this app ,test for the nav, reducer store and the reducer. Please kindly considered my efficiency .thank you sir